config/_default/menus.yaml

```
@@ -51,6 +51,10 @@ main:
     weight: 6
     url: "working-groups/"
     parent: Community
+  - name: "MatNWB"
+    weight: 4
```

I think `weight` is used for ordering and so it should be unique?
```suggestion
    weight: 7
```